src/ontology/lexmatch/split-mapping-set/mondo_exactmatch_icd11.foundation.tsv

```diff
@@ -1,6 +1,7 @@
 subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification	mapping_tool	confidence	subject_match_field	object_match_field	match_string	comment
-MONDO:0000001	disease	skos:exactMatch	icd11.foundation:1659232486	Mitral valve disease, unspecified	semapv:LexicalMatching	oaklib	0.8	oio:hasExactSynonym	oio:hasExactSynonym	disease	LEXMATCH
-MONDO:0000001	disease	skos:exactMatch	icd11.foundation:1659232486	Mitral valve disease, unspecified	semapv:LexicalMatching	oaklib	0.8	rdfs:label	oio:hasExactSynonym	disease	LEXMATCH
+MONDO:0000001	disease	skos:exactMatch	icd11.foundation:1659232486	Mitral valve disease, unspecified	semapv:LexicalMatching	oaklib	0.8	oio:hasExactSynonym	oio:hasExactSynonym	disorder	LEXMATCH
+MONDO:0000001	disease	skos:exactMatch	icd11.foundation:1659232486	Mitral valve disease, unspecified	semapv:LexicalMatching	oaklib	0.8	oio:hasExactSynonym	oio:hasExactSynonym	disorder	LEXMATCH
```

Comment:
Why are these two lines duplicated:
```
MONDO:0000001	disease	skos:exactMatch	icd11.foundation:1659232486	Mitral valve disease, unspecified	semapv:LexicalMatching	oaklib	0.8	oio:hasExactSynonym	oio:hasExactSynonym	disorder	LEXMATCH
```

Reply:
Nice catch. I'll:
- [ ] 1. Try to see if this has happened before or if this is a new issue.
- [ ] 2. Figure out why this is happening, and fix

Reply:
@joeflack4 did you have time to look into the duplicate lines? I don't think it's related, but at the time of reviewing this decided to add the comment here in this PR in the meantime.

Reply:
Nope, not today. It's basically basically the very next thing on my list.

It may not be quick to figure out. If I can't figure it out quickly, I will make an issue. Seems pretty high priority.

It sounds like maybe this PR doesn't need to be held up by that and otherwise this looks good?

Reply:
I looked at a few branches, comment [here](https://github.com/monarch-initiative/mondo-ingest/pull/586#discussion_r1655711141). I'll create a new issue.